terraform/modules/hub/variables.tf

```diff
@@ -53,6 +53,11 @@ variable "zendesk_username" {
   description = "Username (email address) for Zendesk access"
 }
 
+variable "matomo_site_id" {
+  description = "Site ID to use for Matomo"
+  default = 1
```

Comment:
Should we have defaulted this to a non-prod value?

Reply:
We probably should have, but we can also remove the default now `infrastructure-config` is merged 👍